Druid/DRBytecodeObject.class.st

```diff
@@ -0,0 +1,47 @@
+Class {
+	#name : #DRBytecodeObject,
+	#superclass : #DRDispatchTableObject,
```

Comment:
I don't think these should be "dispatch table" objects. Maybe `DRInterpreterInstruction`?